apps/bajour/src/components/baslerin-des-tages/baslerin-des-tages.tsx

```diff
@@ -0,0 +1,206 @@
+import {styled} from '@mui/material'
+import {ApiV1, isImageBlock, isRichTextBlock, isTitleBlock, RichTextBlock} from '@wepublish/website'
+import {useWebsiteBuilder} from '@wepublish/website'
+import {format} from 'date-fns'
+import {de} from 'date-fns/locale'
+
+interface BaslerinDesTagesProps {
+  slug: string
+}
+
+const Headings = styled('div')(({theme}) => ({
+  borderBottom: '1em solid #feddd2',
+  paddingBottom: '0.5em',
+  marginLeft: '3em',
+  display: 'flex',
+  justifyContent: 'space-between',
+  [theme.breakpoints.up('lg')]: {
+    gridColumnStart: 2,
+    paddingBottom: '3em',
+    marginLeft: '1em'
+  }
+}))
+
+const Heading = styled('h1')(({theme}) => ({
+  margin: 0,
+  display: 'block',
+  fontSize: '1.6em',
+  lineHeight: '1.2em',
+  fontWeight: 500
+}))
+
+const Content = styled('section')(({theme}) => ({
+  marginLeft: '3em',
+  marginTop: '1em',
+  gridColumn: '2/3',
+  [theme.breakpoints.up('lg')]: {
+    marginLeft: '1em'
+  }
+}))
+
+const HeadingLarge = styled(Heading)(({theme}) => ({
+  fontSize: '2em',
+  lineHeight: '1.3em',
+  textTransform: 'uppercase'
+}))
+
+const BaslerinDesTagesWrapper = styled('article')(({theme}) => ({
+  overflowX: 'hidden',
+  [theme.breakpoints.up('lg')]: {
+    display: 'grid',
+    gridGap: '1em',
+    gridTemplateColumns: '1fr 2fr',
+    gridTemplateRows: 'minmax(min-content, auto) 1fr'
+  }
+}))
+
+const OnlyMobile = styled('div')(({theme}) => ({
+  display: 'block',
+  [theme.breakpoints.up('lg')]: {
+    display: 'none'
+  }
+}))
+
+const OnlyDesktop = styled('div')(({theme}) => ({
+  display: 'none',
+  [theme.breakpoints.up('lg')]: {
+    display: 'block'
+  }
+}))
+
+const DateDisplay = styled('span')(({theme}) => ({
+  display: 'block',
+  borderBottom: '1px solid black',
+  fontSize: '0.8em',
+  fontWeight: 500,
+  [theme.breakpoints.up('lg')]: {
+    fontSize: '1.3em',
+    textAlign: 'right'
+  }
+}))
+
+const WeekdayDisplay = styled(DateDisplay)(({theme}) => ({
+  border: 'none'
+}))
+
+const Title = styled('span')(({theme}) => ({
+  fontSize: '1.3em',
+  fontWeight: 700,
+  marginBottom: '1em'
+}))
+
+const ImageWrapperMobile = styled(OnlyMobile)(({theme}) => ({
+  display: 'grid',
+  gridTemplateColumns: '50vw repeat(6, 25vw)',
+  gridTemplateRows: '1fr 1fr',
+  gridGap: '0.8em',
```

Comment:
grid-gap is a deprecated css property and superseded by the `gap` property.
We should probably also use the theme spacings here etc